packages/gatsby/src/utils/graphql-typegen/utils.ts

```diff
@@ -14,6 +14,21 @@ export function stabilizeSchema(schema: GraphQLSchema): GraphQLSchema {
   return lexicographicSortSchema(schema)
 }
 
+export function sortDefinitions(
+  a: IDefinitionMeta,
+  b: IDefinitionMeta
+): number {
+  const aKey = a.name
+  const bKey = b.name
+  if (aKey < bKey) {
+    return -1
+  }
+  if (aKey > bKey) {
+    return 1
+  }
+  return 0
```

Comment:
TIL!

Reply:
Should better with `localeCompare(compareString, locales)` because

> If this parameter is not provided, the localeCompare() method will use the host environment's current locale.

Source: https://www.techonthenet.com/js/string_localecompare.php



```suggestion
a.name.localeCompare(b.name,'en')